boilerplate/storybook/storybook.ejs

```diff
@@ -9,5 +10,11 @@ configure(() => {
 // This assumes that storybook is running on the same host as your RN packager,
 // to set manually use, e.g. host: 'localhost' option
 const StorybookUI = getStorybookUI({ port: 7007, onDeviceUI: true })
-AppRegistry.registerComponent('<%= props.name %>', () => StorybookUI)
-export default StorybookUI
+
+class StorybookUIRoot extends Component {
+  render() {
+    return <StorybookUI />
+  }
+}
+AppRegistry.registerComponent('<%= props.name %>', () => StorybookUIRoot)
```

Comment:
You should not have AppRegistry here thanks to the Expo.registerRootComponent in index.js

Reply:
Thanks for reviewing! I've already pushed these changes in the latest commit.